apps/heft/src/plugins/Webpack/WebpackPlugin.ts

```diff
@@ -2,13 +2,19 @@
 // See LICENSE in the project root for license information.
```

Comment:
I tested your PR with my branch from PR https://github.com/microsoft/tsdoc/pull/248 (which converts TSDoc to use Heft). The web server works, but it's printing this warning:

```
[112] ./lib/index.css 1.38 KiB {0} [built]
    + 99 hidden modules

WARNING in asset size limit: The following asset(s) exceed the recommended size limit (244 KiB).
This can impact web performance.
Assets:
  tsdoc-playground.js (312 KiB)

WARNING in entrypoint size limit: The following entrypoint(s) combined asset size exceeds the recommended limit (244 KiB). This can impact web performance.
Entrypoints:
  tsdoc-playground (312 KiB)
      tsdoc-playground.js


WARNING in webpack performance recommendations: 
You can limit the size of your bundles by using import() or require.ensure to lazy load some parts of your application.
For more info visit https://webpack.js.org/guides/code-splitting/
Child html-webpack-plugin for "index.html":
     1 asset
```

The previous implementation did not print this warning. Is it something we can tune?

Reply:
I believe it is tweakable, but I'm surprised it's different between the two. We should look into that seprately.